project_parent/__manifest__.py

```diff
@@ -5,10 +5,9 @@
     "version": "13.0.1.0.0",
     "license": "LGPL-3",
     "category": "project",
-    "author": "Therp B.V., Elico Corp, Odoo Community Association (OCA)",
+    "author": "haulogy SA/NV, Therp B.V., Elico Corp, Odoo Community Association (OCA)",
```

Comment:
I'm not sure there is enough originality to award Authorship.
I would rather have you added to the contributors section, and not here.

Reply:
Sure.